api/v1alpha1/validation/envoyproxy_validate.go

```diff
@@ -152,8 +152,7 @@ func validateBootstrap(boostrapConfig *egv1a1.ProxyBootstrap) error {
 		return fmt.Errorf("dynamic_resources cannot be modified")
 	}
 	// Ensure layered runtime resources config is same
-	if userBootstrap.LayeredRuntime == nil ||
-		cmp.Diff(userBootstrap.LayeredRuntime, defaultBootstrap.LayeredRuntime, protocmp.Transform()) != "" {
+	if cmp.Diff(userBootstrap.LayeredRuntime, defaultBootstrap.LayeredRuntime, protocmp.Transform()) != "" {
```

Comment:
guessing this line isn't needed anymore, since we dont have any existing layered runtime config

Reply:
yes, you're right.